.gitignore

```diff
@@ -10,5 +10,7 @@ build.log
 *.egg-info/
 .coverage
 *~
+.#*
+.*~
```

Comment:
`.*~` This is already covered by `*~`
What kind of tool creates files with `#` in it?